File: Results/DER/Translation/translation_output/w_test/mistral/avatar/java/python/atcoder_ABC051_A.py
<s> [INST] Translate the following java code to python and enclose your solution inside ```python```.
A sample test case is provided below:

Test input:
pnxnn,muduhzn,aquud,

Expected output:
pnxnn asadasd asdasf



```
import java.util.Scanner;

public class atcoder_ABC051_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        scanner.useDelimiter(",");
        
        String a = scanner.next();
        String b = scanner.next();
        String c = scanner.next();
        
        System.out.println(a + " " + b + " " + c);
    }
}
```
[/INST]
```python
import sys

a, b, c = sys.stdin.readline().strip().split()
print(a + " " + b + " " + c)
```</s>

File: Results/DER/Translation/translation_output/misleading_test/starcoder/avatar/python/java-sanitized/atcoder_ABC051_A.java
import java.util.*;
public class atcoder_ABC051_A {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s = sc.nextLine();
        String[] arr = s.split(", ");
        for(int i=0;i<arr.length;i++){
            arr[i] = arr[i].replace(" ", " ");
        }
        String res = "";
        for(int i=0;i<arr.length;i++){
            res += arr[i].substring(0,1).toUpperCase()+arr[i].substring(1,arr[i].length())+" ";
        }
        System.out.println(res.substring(0,res.length()-1));
    }
}